plugins/modules/ndo_physical_interface.py

```diff
@@ -0,0 +1,401 @@
+
+            if physical_interface_type and match.details.get("policyGroupType") != physical_interface_type:
+                mso.fail_json(msg="ERROR: Physical Interface type cannot be changed.")
+
+            if name and match.details.get("name") != name:
+                ops.append(dict(op="replace", path=physical_policy_path + "/name", value=name))
+                proposed_payload["name"] = name
+
+            if description is not None and match.details.get("description") != description:
+                ops.append(dict(op="replace", path=physical_policy_path + "/description", value=description))
+                proposed_payload["description"] = description
+
+            if nodes and match.details.get("nodes") != nodes:
+                ops.append(dict(op="replace", path=physical_policy_path + "/nodes", value=nodes))
+                proposed_payload["nodes"] = nodes
+
+            if physical_policy_uuid and match.details.get("policy") != physical_policy_uuid:
+                ops.append(dict(op="replace", path=physical_policy_path + "/policy", value=physical_policy_uuid))
+                proposed_payload["policy"] = physical_policy_uuid
+
+            if breakout_mode and match.details.get("breakoutMode") != breakout_mode:
+                ops.append(dict(op="replace", path=physical_policy_path + "/breakoutMode", value=breakout_mode))
+                proposed_payload["breakoutMode"] = breakout_mode
+
+            if interfaces and interfaces != match.details.get("interfaces"):
+                ops.append(dict(op="replace", path=physical_policy_path + "/interfaces", value=interfaces))
+                proposed_payload["interfaces"] = interfaces
+
+            # Node changes are not reflected on UI
+            if interface_descriptions and match.details.get("interfaceDescriptions") != interface_descriptions:
+                updated_interface_descriptions = format_interface_descriptions(interface_descriptions, "")
+                ops.append(dict(op="replace", path=physical_policy_path + "/interfaceDescriptions", value=updated_interface_descriptions))
+                proposed_payload["interfaceDescriptions"] = updated_interface_descriptions
+            elif interface_descriptions == [] and match.details.get("interfaceDescriptions"):
+                ops.append(dict(op="remove", path=physical_policy_path + "/interfaceDescriptions"))
+
+            mso.sanitize(proposed_payload, collate=True)
+
+        else:
+            if not nodes:
+                mso.fail_json(msg=("ERROR: Missing 'nodes' for creating a Physical Interface."))
+
+            if not physical_interface_type:
+                mso.fail_json(msg=("ERROR: Missing Physical Interface type for creating a Physical Interface."))
+
+            payload = {
+                "name": name,
+                "nodes": nodes,
+                "interfaces": interfaces,
+                "policyGroupType": physical_interface_type,
+            }
```

Comment:
```suggestion
            mso_values = dict(
                name = name,
                nodes = nodes,
                interfaces = interfaces,
                policyGroupType = physical_interface_type,
            )
```
Our team planned to maintain the variable naming consistency for the new modules.

Reply:
changed already